content/en/network_monitoring/devices/snmp_metrics.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: Network Device Monitoring Setup
+title: SNMP Metrics
```

Comment:
Change title to "Collect SNMP Metrics from your Devices" 

This is still the general setup for the product, e.g you must do this to get any of the UI to work, but I want to be clear that this setup does not include sending Traps or Netflow hence the separation, so I think it's best to add more to the tile